### DISCUSSION TITLE
How to organize my projects?

### DISCUSSION THREAD

**Discussion by jonatiao** [DependencyTrack/dependency-track#2229] [Q&A]
I have several projects, each project may have, for instance, the following classifiers:
Container
Application
Operating System

Currently we are creating projects names like:
myproject-container
myproject-application
myproject-os

I would like if there's any plan in the future to support multiple BOMs classifiers for the same project.
Or maybe we are not organizing our projects correctly.. any tips : )

thx everyone!!

**Comment by malice00** [comment]
Maybe you can use one of the tools from here: https://cyclonedx.org/tool-center/ to merge your BOMs together? Eg with https://github.com/CycloneDX/cyclonedx-cli ?

I have to admit though, I haven't tried it myself (yet)...

**Reply by syalioune** [reply]
Merging is indeed a solution to avoid myriad of projects (that the approach i'm using) but you lose accountability if the sub-projects are managed by different teams. Another alternative since v4.7 is the project parent-child relationship feature (See https://github.com/DependencyTrack/dependency-track/pull/2152). 
You can then create one parent project (i.e myproject) and child projects (os, container, app) which will inherits various config from the parent (notifications, policies,...)

**Comment by meightymike** [comment]
I realize this is an older thread, but I'm new to Dependency-Track and currently exploring the best way to organize our company's projects efficiently.

Our teams primarily use NuGet and npm packages. I'm considering a structure similar to what @jonatiao described, and would love to hear your thoughts on best practices for managing projects.

- projectname  
  - projectname - npm  
    - projectname - npm - solution.A 
    - projectname - npm - solution.B  
    - ...  
  - projectname - nuget  
    - projectname - nuget - solution.B  
    - projectname - nuget - solution.C  
    - ...
